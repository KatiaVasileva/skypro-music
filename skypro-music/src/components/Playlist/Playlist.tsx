import { Track } from "@/types/Track.types";
import styles from "./Playlist.module.css";
import TrackItem from "../Track/Track";

function Playlist({allTracks} : {allTracks : Array<Track>}) {
    return (
        <div className={styles.playlistContent}>
        {allTracks.map((track) => (
          <TrackItem track={track} key={track._id} tracks={allTracks} />
        ))}
      </div>
    );
}

export default Playlist;