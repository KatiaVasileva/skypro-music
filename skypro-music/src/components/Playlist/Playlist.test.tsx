import ReduxProvider from "@/store/ReduxProvider";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Playlist from "./Playlist";

describe("Playlist component", () => {
  it("renders correctly", () => {
    const { container } = render(
      <ReduxProvider>
        <Playlist allTracks={[]} />
      </ReduxProvider>
    );
    expect(container).toMatchSnapshot();
  });
});