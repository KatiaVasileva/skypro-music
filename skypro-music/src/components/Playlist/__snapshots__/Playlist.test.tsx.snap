// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playlist component renders correctly 1`] = `
<div>
  <div
    class="playlistContent"
  />
</div>
`;
